framework/source/class/qx/ui/form/renderer/AbstractRenderer.js

```diff
@@ -51,6 +51,31 @@ qx.Class.define("qx.ui.form.renderer.AbstractRenderer",
     form.addListener("change", this._onFormChange, this);
   },
 
+  properties :
+    {
+      /**
+       * A string that is appended to the label if it is not empty
+       */
+      labelSuffix :
+        {
+          check : "String",
+          init : " :",
+          event : "changeLabelSuffix",
+          nullable : true
+        },
+
+      /**
+       * A string that is appended to the label if it is not empty
```

Comment:
This documentation string is the same as the one above. Both properties have a bit more (and different) detail about what the property actually does.